geonode/documents/views.py

```diff
@@ -531,7 +531,7 @@ def document_list(request):
                          document.hazard_type,
                          other_perils,
                          document.hazard_unit,
-                         document.distribution_url,
+                         document.detail_url,
```

Comment:
There is 18 labels on header for 19 fields of data. Can you add a header for detail_url field ?